gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -133,7 +160,9 @@ private void addNewObservation(Feature feature, Form form) {
     navigator.navigate(HomeScreenFragmentDirections.addObservation(projectId, featureId, formId));
   }
 
-  /** This is only possible after updating the location of the feature. So, reset the UI. */
+  /**
+   * This is only possible after updating the location of the feature. So, reset the UI.
```

Comment:
This method docs doesn't give enough context to understand this statement.  Also, "reset the UI" is misleading; the entire UI state isn't wiped out, only the current edit mode. Please update to include a) when this method is called and b) what it does.